routes/_routes.js

```diff
@@ -16,17 +19,96 @@ router.get('/', function (req, res) {
     });
 });
 
+/*
+ * registration routes
+ */ 
+
 router.get('/register', function (req, res) {
-    res.redirect('/users/new');
+    res.render('register', {
+        appName: appName,
+        pageTitle: "Register",
+    });
+});
+ 
+
+router.post('/register', function(req, res) {
```

Comment:
RESTful route would be POST /users, so we would probably move this to the users route

Reply:
we will do this later